tests/stdlib/types/collections/torderedtables_regression.nim

```diff
@@ -0,0 +1,57 @@
+discard """
+  description: "A regression test for lazily initialized `OrderedTable`s"
+  target: native

```

Comment:
```suggestion
  target: native
  timeout: "10"
```

We're testing an infinite loop, I figure an explicit timout make sense? 10 might be too high.

Reply:
Only a single iteration is expected, so an assertion is raised if a second one happens. In other words, the infinite loop is prevented and no intervention from testament's side is required.

Reply:
Got it, thanks.